### DISCUSSION TITLE
A <video> won't autoplay

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#772] [Report a bug] [answered]
I am not sure if this is the right place to ask. Maybe smth like Stackexchange is a more suitable platform. But still --

I was trying to embed two videos on my homepage so that they autoplay and repeat. Normally, the modern browsers' policy is that only muted video may autoplay, so I included the `muted` tag as well.

```
<video autoplay muted loop>
  <source src="/media/images/video.mp4" type="video/mp4" />
  Your browser does not support the video tag.
</video>
```

However, the videos just won't autoplay. I tried two browsers, Vivaldi and Falkon. Maybe you, guys, know what I was doing wrong?

P. S. I tried using [a video from this seemingly reliable website](https://www.w3schools.com/html/html5_video.asp) but it still did not autoplay.

**Comment by markseuffert** [comment, marked as answer]
Thanks for your bug report. I updated the [core extension](https://github.com/annaesvensson/yellow-core). The [video element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) was missing two attributes in the HTML filter. You can tell whether something is filtered out by looking at the generated HTML code in the browser.

Should be fixed now, please try it out.

**Reply by glow12121** [reply]
Many thanks, Mark, it works now!

If I have the time, I will look into Yellow's extensions and try to make one for videos.
